app/scripts/models/auth_brokers/oauth.js

```diff
@@ -68,8 +68,12 @@ function (_, Constants, Url, OAuthErrors, AuthErrors, p, Validate,
       this._assertionLibrary = options.assertionLibrary;
       this._oAuthClient = options.oAuthClient;
 
-      return BaseAuthenticationBroker.prototype.initialize.call(
+      var result = BaseAuthenticationBroker.prototype.initialize.call(
                   this, options);
+
+      this.unsetCapability('interTabSignIn');
```

Comment:
should this override this in the same way as it is done here instead of using `unset`: https://github.com/mozilla/fxa-content-server/blob/master/app/scripts/models/auth_brokers/fx-fennec-v1.js#L27 ? Also why is OAuth not using interTabSignIn, could we add a comment?